include/oxen/quic/gnutls_crypto.hpp

```diff
@@ -303,6 +384,7 @@ namespace oxen::quic
         gnutls_anti_replay_t anti_replay;
 
         bool is_client;
```

Comment:
Not in this PR, but just noticed this member isn't `_`-prefixed for some reason, while all the other ones are.  Might be a nice change to make it match.

Reply:
My mistake, thank you!